### PR TITLE
Move quorum out of verifyVM

### PR DESCRIPTION
**Pull request by claudijd**
---
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1162)

### CODE REVIEW

File: ethereum/contracts/Messages.sol [hunk 1]
```diff
@@ -51,7 +51,8 @@ contract Messages is Getters {
         *   if making any changes to this, obtain additional peer review. If guardianSet key length is 0 and 
         *   vm.signatures length is 0, this could compromise the integrity of both vm and signature verification.
         */
-        if(((guardianSet.keys.length * 10 / 3) * 2) / 10 + 1 > vm.signatures.length){
+        (bool quorum_result) = quorum(vm.signatures, guardianSet);
+        if !quorum_result{
```
[the remark this answers is not in the document]
Comment (claudijd) [reply ethereum/contracts/Messages.sol:55]: ok, will clean that up

File: ethereum/contracts/Messages.sol [hunk 2]
```diff
@@ -51,7 +51,7 @@ contract Messages is Getters {
         *   if making any changes to this, obtain additional peer review. If guardianSet key length is 0 and 
         *   vm.signatures length is 0, this could compromise the integrity of both vm and signature verification.
         */
-        if(((guardianSet.keys.length * 10 / 3) * 2) / 10 + 1 > vm.signatures.length){
+        if (vm.signatures.length > quorum(guardianSet.keys.length)){
```
Comment (kcsongor) [inline ethereum/contracts/Messages.sol:54]: shouldn't this be `<`?

File: ethereum/contracts/Messages.sol [hunk 3]
```diff
@@ -145,4 +145,11 @@ contract Messages is Getters {
 
         vm.payload = encodedVM.slice(index, encodedVM.length - index);
     }
+
+    /**
+     * @dev quorum serves solely to determine the number of signatures required to acheive quorum
+     */
+    function quorum(uint numGuardians) public pure virtual returns (uint numSignaturesRequiredForQuorum) {
+        return ((numGuardians * 2) / 3) + 1;
```
Comment (claudijd) [inline ethereum/contracts/Messages.sol:153]: @tbjump @kcsongor FYI

**Review by tbjump — APPROVED** [review]
looks good

**Pull request merged**